client/components/tinymce/plugins/simple-payments/dialog/product-image.jsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import MediaUtils from 'lib/media/utils';
+import QueryMedia from 'components/data/query-media';
+import { getMediaItem } from 'state/selectors';
+
+const ProductImage = ( { siteId, imageId, image } ) => {
+	if ( ! siteId || ! imageId ) {
+		return null;
+	}
+
+	if ( ! image ) {
+		return (
+			<figure className="editor-simple-payments-modal__list-figure is-placeholder">
```

Comment:
In fact why we are rendering an empty DOM element. is it just to load the image through of the querydata component?